01-python/outdoors/lake.py

```
@@ -1,3 +1,36 @@
-def draw_lake():
-    print("lake not found")
-    return
\ No newline at end of file
+from random import random
+
+def draw_lake(numFish=3):
+  # Make sure user didn't put in a number that was too high
+  numFish = 8 if numFish > 8 else numFish
```

it's better to raise an error or notify the user somehow that you are changing their input. Otherwise they will be debugging for ages trying to understand why it is not printing > 8 fish